mission-control/docs/config-db/concepts/insights.md

```diff
@@ -0,0 +1,15 @@
+# Insights
+
+Some scrapers have the ability to tap into an advisor source that provides insights about a config.
```

Comment:
Scrapers can create and attach insights (security, performance, cost, etc..) to related config items.